experiments/opencog/cog_module/cognets/module.py

```diff
@@ -46,7 +46,7 @@ def set_value(atom, value, tv=False):
         assert isinstance(value, TTruthValue) or isinstance(value, CogModule)
     else:
         key = atom.atomspace.add_node(types.PredicateNode, "cogNet")
-    atom.set_value(key, PtrValue(value))
+    atom.set_value(key, ptr)
```

Comment:
Looks like `ptr` is not defined in this context, should it be `PtrValue(value)` as before?

Reply:
Yes